rdt/hyper_transformer.py

```diff
@@ -189,6 +189,36 @@ def update_field_data_types(self, field_data_types):
         self.field_data_types.update(field_data_types)
         self._unfit()
 
+    def update_transformers_by_sdtype(self, sdtype, transformer):
```

Comment:
Can we also delete the `update_default_data_type_transformers` method? I think this is essentially replacing that functionality